magenta/models/onsets_frames_transcription/onsets_frames_transcription_create_dataset.py

```diff
@@ -257,8 +85,8 @@ def generate_train_set(exclude_ids):
       # load the midi data and convert to a notesequence
       ns = midi_io.midi_file_to_note_sequence(pair[1])
 
-      splits = find_split_points(ns, samples, FLAGS.sample_rate,
-                                 FLAGS.min_length, FLAGS.max_length)
+      splits = create_dataset_util.find_split_points(
```

Comment:
Why did this need to get moved?

Reply:
The test was importing this file, which has flags in it. Another test is also doing this and there was a conflicting flag name. Apparently pytest doesn't clear these globals between tests. Also, testing functions from a binary file is a no-no in general (partially for this reason).